spec/models/reading_spec.rb

```diff
@@ -41,5 +41,21 @@
       })
       expect(@reading.temp).to eq 57
     end
+
+    it "handles nil temperature properly", :vcr do
+      allow(WeatherMan).to receive(:outdoor_temp_for).and_return(nil)
+      user = create(:user, zip_code: '10216')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.